cmd/vulcan-wpscan/wpscan.go

```diff
@@ -162,6 +166,33 @@ type Vulnerability struct {
 	} `json:"references"`
 }
 
+func logAPITokenStatus(token string, l *logrus.Entry) {
```

Comment:
I agree completely. After further consideration, it may not be necessary to include this metric.
If the execution fails due to API limits being reached, the check will already indicate this in the error message and mark it as a failure. This should provide us with sufficient information for evaluation purposes.
Function removed in [5593c64](https://github.com/adevinta/vulcan-checks/pull/308/commits/5593c640d8826dc632ebcb5f1bb67180389b1537).